docs/account/delete-account.mdx

```diff
@@ -3,17 +3,19 @@ title: "How do I delete my account?"
 description: "Instructions on deleting your CodeCrafters account"
 ---
 
-To delete your CodeCrafters account and all associated data, please write to us at `hello@codecrafters.io`.
+To delete your CodeCrafters account and all associated data, follow these steps:
 
-Here's a template you can use:
-
-<Card title="Click to send email" icon="envelope" iconType="duotone" href="mailto:hello@codecrafters.io?subject=Account deletion request">
-  **To**: hello@codecrafters.io
-
-**Subject**: Account deletion request
-
-**Body**:
-
-Please delete my CodeCrafters account. My GitHub username is `your_github_username`.
-
-</Card>
+1. Click on your username on the top right corner of the page.
```

Comment:
Maybe we can make this simpler: 

> Visit your [account settings page](https://app.codecrafters.io/settings/account)

Reply:
And the other things we can cover can be like FAQ - what happens when you delete your account, can you signup again etc?